Makefile

```diff
@@ -13,7 +13,7 @@ deps-update:
 
 lint:
 	yapf $(analysis_directories) --diff --parallel --recursive --style google
-	bandit -r $(analysis_directories)
+	bandit -r $(analysis_directories) --skip B101
```

Comment:
What check is `B101`? Could you add a comment explaining why/what is being skipped, or use a long form name if that's possible (like in pylint)?

Reply:
Long-form isn't possible with bandit, but I can add a comment. It's just warning about the "assert" statements, which are ignored when compiling Python into bytecode